lib/Sema/DerivedConformanceError.cpp

```diff
@@ -22,11 +22,40 @@
 #include "swift/AST/Expr.h"
 #include "swift/AST/Module.h"
 #include "swift/AST/Types.h"
+#include "swift/AST/SwiftNameTranslation.h"
 
 using namespace swift;
+using namespace swift::objc_translation;
 
 static void deriveBodyBridgedNSError_enum_nsErrorDomain(
-              AbstractFunctionDecl *domainDecl, void *) {
+                    AbstractFunctionDecl *domainDecl, void *) {
+  // enum SomeEnum {
+  //   @derived
+  //   static var _nsErrorDomain: String {
+  //     return _typeName(self, qualified: true)
```

Comment:
Should this not be `return String(reflecting: self)`?

Reply:
Ack, you're right. Artifact of this PR's convoluted history.